### ISSUE TITLE
The app name in .platform.app.yaml is confusing

### ISSUE THREAD

**Issue by jcnventura** [platformsh/platformsh-examples#40]
The default app name in all .platform.app.yaml examples (for Drupal 7 at least) is simply "php".

This leads to confusion when specifying the route in .platform/routes.yaml which by default is simply:
php:php

If the example default would be changed to 'appy', then it would be simple to understand what the correct route is by reading the example as appy:php, and not php:appy.

**Comment by GuGuss** [comment]
I agree with you on that. We need to change our snippets in the documentation too so it matches the examples.

:+1: for this.
